lib/pages/manage_account.dart

```diff
@@ -430,6 +430,7 @@ class _ManageAccount extends HookWidget {
           ),
           child: const Text('DELETE ACCOUNT'),
         ),
+        const SafeArea(child: SizedBox.shrink()),
```

Comment:
Please make sure it is actually needed, since it is in a `Scaffold` it shouldn't be needed